moduleroot/.github/workflows/ci.yml.erb

```diff
@@ -0,0 +1,106 @@
+name: CI
+
+on:
+  pull_request:
+  schedule:
+    - cron: '4 4 * * *'
+
+jobs:
+  setup_matrix:
+    if: github.event_name != 'schedule' || github.repository_owner == '<%= @configs[:namespace] %>'
+    name: 'Setup Test Matrix'
+    runs-on: ubuntu-latest
+    outputs:
+      beaker_setfiles: ${{ steps.get-outputs.outputs.beaker_setfiles }}
+      puppet_major_versions: ${{ steps.get-outputs.outputs.puppet_major_versions }}
+      puppet_unit_test_matrix: ${{ steps.get-outputs.outputs.puppet_unit_test_matrix }}
+    env:
+      BUNDLE_WITHOUT: development:release
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: '2.7'
+          bundler-cache: true
+      - name: Run rake check
+        run: bundle exec rake check
+      - name: Run rake validate
+        run: bundle exec rake validate
+      - name: Run rake lint
+        run: bundle exec rake lint
+      - name: Setup Test Matrix
+        id: get-outputs
```

Comment:
call me weird, but I always find `-` in names/ids confusing

Reply:
I think I copied that from pdk-templates. Not sure what a better name would be.

Reply:
`get_outputs`? :)